src/main/java/com/rtb/projectmanagementtool/project/ProjectTestServlet.java

```diff
@@ -0,0 +1,37 @@
+// Just using this Servlet to test the Project & ProjectController classes.
+// I didn't know any other ways to test the classes other than through unit tests, which I
+// couldn't get working.
+// Feel free to mess around with it and test some of the functionality/see how it can be improved
+
+package com.google.sps.servlets;
```

Comment:
Please watch out for package name vs. directory structure.  In most cases they would have to be the same.